### PR TITLE
separete out metrics/darwin/swap.go from memory.go

### PR DESCRIPTION
**Pull request by Songmu**
Improve the codes around memory metrics collectors on osx based on mechairoi's feedbacks at #84.

### CODE REVIEW

**Comment by Songmu** [comment]
@mechairoi could you review it?

File: metrics/darwin/swap.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+)
+
+/*
+SwapGenerator collect swap usage
+
+`memory.{metric}`: using swap size retrieved from `sysctl vm.swapusage`
+
+metric = "swap_total", "swap_free"
+
+graph: stacks `memory.{metric}`
```
Comment (mechairoi) [inline metrics/darwin/swap.go]: [nit] swap metrics are not stacked?

**Comment by mechairoi** [comment]
[nit] Please fix comments at
https://github.com/mackerelio/mackerel-agent/blob/fix/mac-memory/metrics/darwin/memory.go#L20

**Comment by mechairoi** [comment]
LGTM!

**Pull request merged**